tools/check-typo-since

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+#**************************************************************************
+#*                                                                        *
+#*                                 OCaml                                  *
+#*                                                                        *
+#*            Gabriel Scherer, projet Parsifal, INRIA Saclay              *
+#*                                                                        *
+#*   Copyright 2018 Institut National de Recherche en Informatique et     *
+#*     en Automatique.                                                    *
+#*                                                                        *
+#*   All rights reserved.  This file is distributed under the terms of    *
+#*   the GNU Lesser General Public License version 2.1, with the          *
+#*   special exception on linking described in the file LICENSE.          *
+#*                                                                        *
+#**************************************************************************
+
+# Run check-typo, comparing only files that have changed since a particular
+# git state
+
+check_typo_since() {
+  CHECK_TYPO=$(dirname $0)/check-typo
+  for f in $(git diff --name-only $1)
```

Comment:
Filename processing should be done in a `while` loop - this will break weirdly if anyone commits a filename with a space. See the `CheckTypoTree` function in `travis-ci.sh` (although I discover that I forgot to quote `$path` in that function 😊)

Reply:
I'll look at this, thanks.